app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -199,10 +199,10 @@ class BookmarkFragment : LibraryPageFragment<BookmarkNode>(), BackHandler {
                 metrics?.track(Event.OpenedBookmarksInPrivateTabs)
                 true
             }
-            R.id.edit_bookmark_multi_select -> {
+            R.id.share_bookmark_multi_select -> {
                 val bookmark = bookmarkStore.state.mode.selectedItems.first()
                 navigate(
-                    BookmarkFragmentDirections.actionBookmarkFragmentToBookmarkEditFragment(
+                    BookmarkFragmentDirections.actionBookmarkFragmentToShareFragment(
                         bookmark.guid
```

Comment:
The url, title, and tabs array need to be passed as arguments. See `HistoryFragment` for an example.